tsconfig.extensions.json

```diff
@@ -1,11 +1,9 @@
 {
   "extends": "./tsconfig.json",
-  "files": [
-    "src/extensions/extension-api.ts",
-  ],
   "compilerOptions": {
     "module": "CommonJS",
-    "sourceMap": false,
-    "declaration": true
+    "declaration": true, // should output .d.ts
+    "sourceMap": true, // for use with webpack's devtool: 'inline-source-map'
```

Comment:
Comments are not valid here and editors/IDE mostly highlight with an error..

Reply:
yeah, not sure why VSCode didn't warn me :D